components/sendsms/actions/send-message-with-unsubscribe-link/send-message-with-unsubscribe-link.mjs

```diff
@@ -25,12 +24,33 @@ export default {
       description: "The sender ID that appears on the recipient's device.",
       optional: true,
     },
-    report_mask: {
+    reportMask: {
       type: "integer",
       label: "Report Mask",
       description: "Reporting options for delivery status.",
       optional: true,
-      default: 19,
+      options: [
+        {
+          label: "Delivered",
+          value: 1,
+        },
+        {
+          label: "Undelivered",
+          value: 2,
+        },
+        {
+          label: "Queued at network",
+          value: 4,
+        },
+        {
+          label: "Sent to network",
+          value: 8,
+        },
+        {
+          label: "Failed at network",
+          value: 16,
+        },
```

Comment:
I'd put the prop `reportMask` in the app file so you don't have to list the options in more than one place.